script.js

```diff
@@ -21,3 +50,107 @@ document.querySelectorAll('.navbar-links').forEach((n) => {
     heroLogoColor.classList.remove('active');
   });
 });
+
+let p;
+let title;
+let subTitle;
+let text;
+let tech;
+let techUl;
+let subTitleUl;
+
+const container = document.querySelectorAll('.container');
+
+for (let i = 0; i < cardContainer.length; i += 1) {
+  p = container[i];
+
+  title = document.createElement('h2');
+  title.classList.add('project-content-2');
+  title.textContent = cardContainer[i].title;
+  title = title.outerHTML;
+
+  text = document.createElement('p');
+  text.classList.add('project-content-4');
+  text.textContent = cardContainer[i].text;
+  text = text.outerHTML;
+
+  // Add Languages
+
+  techUl = document.createElement('ul');
+  techUl.classList.add('languages-layout');
+
+  for (let j = 0; j < cardContainer[i].languages.length; j += 1) {
+    tech = document.createElement('li');
+    tech.classList.add('languages-list');
+    tech.textContent = cardContainer[i].languages[j];
+    techUl.appendChild(tech);
+  }
+
+  // Add SubTitle
+
+  subTitleUl = document.createElement('ul');
+  subTitleUl.classList.add('project-content-3');
+
+  let counter = 0;
+  let SubTitleCounter;
+
+  for (let k = 0; k < cardContainer[i].SubTitle.length; k += 1) {
+    subTitle = document.createElement('li');
+    SubTitleCounter = document.createElement('img');
+    SubTitleCounter.src = 'img/counter.svg';
+
+    subTitle.appendChild(SubTitleCounter);
+    subTitle.textContent = cardContainer[i].SubTitle[k];
+
+    SubTitleCounter.style = 'padding-right:10px;';
+
+    if (counter === 0) {
+      subTitle.classList.add('title-element');
+      subTitleUl.appendChild(subTitle);
+      counter += 1;
+    } else {
+      subTitle.classList.add('list-element');
+      subTitleUl.appendChild(SubTitleCounter);
+      subTitleUl.appendChild(subTitle);
+    }
+  }
+
+  p.innerHTML = `
+      <button type="button" data-bs-toggle="modal" data-bs-target="#myModal-${i}">See Project</button>
+      <div class="modal" style="display:none" id="myModal-${i}">
+        <div class="modal-dialog modal-lg">
+          <div class="modal-content">   
+            <div class="first-popup-content" style="padding:20px;">
+              ${title}
+              <br>
+              ${subTitleUl.outerHTML}
+              <br>
```

Comment:
- [ ] Kindly use css to add margin or padding instead of using `br` this will make your project more semantic, It would be better to remove all `<br>`  from your project 👍🏿